doc/AppMgmt.xml

```diff
@@ -268,9 +268,34 @@
           </varlistentry>
         </variablelist>
       </section>
+      <section>
+        <title>Backup and Restore</title>
+        <para>A device signals support for backup and restore of individual applications by
+          providing the Configuration URI int the application information items (see <xref
+            linkend="_AppInfo"/>). Clients can retrieve and backup the current configuration via the
+          http GET method. Note that the configuration itself is an opache blob. A backed-up
+          configuration may be restored at a later time by uploading it to the device via the http
+          POST method. Refer to the individual application documentation regarding support for
+          restoring of configurations across devices and versions. For general error behavior of the
+          restore function see <xref linkend="_Ref4595104"/>. Additionally an application may generate 
+          the following error response</para>
+        <itemizedlist>
+          <listitem>
+            <para>521 Version mismatch The application cannot restore the provided configuration.</para>
+          </listitem>
+        </itemizedlist>
+      </section>
+      <section>
+        <title>Application Interface</title>
+        <para>A device should provide an interface description for applications providing a programming API 
+          for configuration and monitoring purposes. For these applications a device should provide a
+          link via the InterfaceDescription information item as defined in <xref linkend="_AppInfo"/>. It allows
+          clients to easily integrate application configuration and information retrieval. Examples for 
+        interface description languages are JSON or YAML descriptions according to OpenAPI.</para>
```

Comment:
I think it should be a URI to the UI itself not the description of the UI. This will make it easy for the VMS just to link to the UI for configuration of the App. Having the VMS understand the inner workings of the App's configuration is too much.

Reply:
This link is about a programmable interface. A link to the UI would also make sense, but from my point of view that would need its own entrypoint.

Reply:
For the webui of app, we could reuse the existing parameter.  
 Website - "Link to supplementary information about the application or its vendor"

Reply:
I think there will be little interest in this API definition URI, especially if it's unclear as above. "Examples for interface description languages are JSON or YAML descriptions according to OpenAPI". It can be anything? What is the VMS going to do with this? It can be OpenAPI? It can be a binary blob that you have to install? I think this belongs in vendor extenstions, and put the URI for the UI in the standardized interface.